logger/src/lib.rs

```diff
@@ -112,7 +124,7 @@ pub fn init() {
             Ok(s) => parse_env(&s),
             Err(_) => Vec::new(),
         };
-        let config = config_console_appender(directives);
+        let config = config_console_appender("", directives);
```

Comment:
Why don't refactor `init()` as `init_console()` directly?

Reply:
I want to amend less code. if change the name of `init()` ，caller should changed.

Reply:
`logger::init();`  is used only https://github.com/ouwenkg/cita/blob/reorganize_services_toml_path/tools/relayer-parser/src/main.rs#L52
So, you could have a try.

Reply:
I like strict you.  I will replace `init()` as `init_console()`.

Reply:
I find this is better: 
```
pub fn init() {
     init_console("");
}
```
`init()` used for tests. no need change other code.